.github/workflows/build.yml

```diff
@@ -15,8 +15,30 @@ jobs:
         with:
           fetch-depth: 0
           persist-credentials: false
+      - name: Cache node modules
```

Comment:
I'd add a comment here pointing out what @ajperezbau and the github actions docs say regarding caching `node_modules`. I know right now we are not using the `setup-node` action, as the docker image provided by `cypress` that we are using already has node preinstalled in it. Nonetheless, since we might totally change how we manage and/or hoist our dependencies (maybe start using `pnpm` instead of `npm` + `lerna bootstrap --hoist`) and might also end up replacing `cypress` in favour of `playwright`, I'd state the good practice just for our future selves.